webapp/packages/plugin-data-viewer/src/ContainerDataSource.ts

```diff
@@ -221,8 +219,7 @@ export class ContainerDataSource extends ResultSetDataSource<IDataContainerOptio
   }
 
   async dispose(): Promise<void> {
-    await this.closeResults(this.results);
-    await this.executionContext?.destroy();
+    await Promise.all([this.closeResults(this.results), this.cancel(), super.dispose()]);
```

Comment:
execution close results after disposing execution context will lead to error, please check that we maybe shouldn't close results in case we destroying context

Reply:
just remove this fn